packages/api/src/objectSet/EXPERIMENTAL_ObjectSetListener.ts

```diff
@@ -47,6 +47,6 @@ type ObjectUpdate<
   O extends ObjectOrInterfaceDefinition,
   P extends PropertyKeys<O>,
 > = {
-  object: Osdk.Instance<O, never, P>;
+  object: Osdk.Instance<O, "$notStrict", P>;
```

Comment:
I want all properties to be `| undefined` since it's possible if you get a reference update, you won't get an updated value